packages/contracts/tasks/migrations/sidechain.mainnet.ts

```diff
@@ -46,9 +46,16 @@ task('sidechain:mainnet', 'Deploy market at sidechain')
     await DRE.run('full:initialize-lending-pool', { pool: POOL_NAME });
 
     console.log("6.1. Initialize lending pool tranche 0");
-    await DRE.run("full:initialize-lending-pool-tranches-0-OP", {
-      pool: POOL_NAME,
-    });
+    if(POOL_NAME=="Optimism") {
+      await DRE.run("full:initialize-lending-pool-tranches-0-OP", {
+        pool: POOL_NAME,
+      });
+    }
+    if(POOL_NAME=="Base") {
```

Comment:
use else if and have an else statement that fails in case we mess something up. makes deployments safer